Vizzuality/trase
# Update country coordinates from restcountries.eu

## Description

France had wrong coordinates on the expore page. Now we get them from https://restcountries.eu/

## Testing instructions

Go to explore page and select Soy and Brazil. France line should go correctly to the France position

![image](https://user-images.githubusercontent.com/9701591/90755302-474a7a80-e2db-11ea-8d8f-cad9619ec0bf.png)
